front/src/assets/css/main.css

```diff
@@ -13,20 +18,9 @@ a {
 	text-decoration: none;
 }
 
-@font-face {
```

Comment:
So we downloaded the fonts, but didn't end up using them in our project.  I would prefer to use exclusively the downloaded fonts, reducing page load.  Only include the fonts you think you will use.